Library/Formula/wxmac.rb

```diff
@@ -15,6 +15,8 @@ class Wxmac < Formula
   depends_on "jpeg"
   depends_on "libpng"
   depends_on "libtiff"
+  
+  option "with-enable-stl", "use standard C++ classes for everything"
```

Comment:
Ok, cool, makes sense to leave it as-is then (but just change the name to `with-stl`)